bot/cogs/fun.py

```diff
@@ -159,7 +159,7 @@ async def coin_toss(self, ctx: Context, toss: str) -> None:
                     "`quantity` - how many to roll\n"
                     "`sides` - how many sides each die will have",
     )
-    async def dice(self, ctx: Context, n: int, sides: int) -> None:
+    async def dice(self, ctx: Context, quantity: int = 1, sides: int = 6) -> None:
```

Comment:
You have changed the parameters of this command, but the code block still is using the previous command signature

`n is not defined`